entity-service-client/src/main/java/org/hypertrace/entity/data/service/client/EdsCacheClient.java

```diff
@@ -101,8 +101,10 @@ public Entity getByTypeAndIdentifyingAttributes(String tenantId,
     try {
       return getById(tenantId, entityIdsCache.get(key));
     } catch (ExecutionException e) {
-      LOG.warn("Failed to fetch entity of tenantId: {}, entityId: {}",
-          key.tenantId, key.byTypeAndIdentifyingAttributes, e);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Failed to fetch entity of tenantId: {}, entityId: {}",
+                key.tenantId, key.byTypeAndIdentifyingAttributes, e);
```

Comment:
```suggestion
                key.tenantId, key.byTypeAndIdentifyingAttributes);
```

Reply:
I think stacktrace isn't printed for debug logs. So better get rid of that argument.